source/playbooks/setting-up-playbooks.rst

```diff
@@ -42,39 +42,23 @@ Defining actions
 
 .. image:: ../images/Playbook-keyword-monitoring.png
    :alt: Keyword notification.
 
-When a run starts
-~~~~~~~~~~~~~~~~~
+If you find your keywords result in too many false positives, consider refining your list and also consider that URLs used by run members may also contain monitored keywords.
 
-  * **Invite members**: Enable the toggle, and select members who will automatically be invited to the channel when the playbook run starts.
-  * **Assign the owner role**: Enable the toggle and select a member. This member is automatically assigned as the owner of the playbook run.
-  * **Announce in another channel**: Enable the toggle and select a channel. When the playbook run is started, an announcement is made in the selected channel.
-  * **Send outgoing webhook**: Enable the toggle and enter the webhook you want to use for when the run starts. For information about the webhook payload, see the `PlaybookRunWebhookPayload struct <https://github.com/mattermost/mattermost-plugin-playbooks/blob/b4c8058d8660efe35050bc7eb080e3819c7ab09c/server/app/playbook_run_service.go#L176-L185>`_. An example of the JSON payload for a run start `is available here <https://gist.github.com/icelander/b68f2bf2b4ffefec93400cb050211cf1>`_.
+Webhooks
+~~~~~~~~
 
-When a status update is posted
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-  * **Send outgoing webhook**: Enable the toggle and enter the webhook you want to use for status updates. For information about the webhook payload, see the `PlaybookRunWebhookPayload struct <https://github.com/mattermost/mattermost-plugin-playbooks/blob/b4c8058d8660efe35050bc7eb080e3819c7ab09c/server/app/playbook_run_service.go#L176-L185>`_. An example JSON payload for a status update `is available here <https://gist.github.com/icelander/2f9938ad68d1e0aa656f97969895d080>`_.
-  
-When a new member joins the channel
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
- * **Send a welcome message**: Enable the toggle and enter text that new channel members will see when they join the channel.
- * **Add the channel to a sidebar category**: Enable the toggle so that the channel will be automatically categorized and listed under **Playbook runs** in the left-hand sidebar.
- 
-When a run is archived
-~~~~~~~~~~~~~~~~~~~~~~
-
- * **Export channel**: (E20 and Cloud only) When a run finishes, this triggers the playbook to automatically save a copy of all channel messages in CSV format. This provides a record of the discussion for future reference or other use such as audits.
+- For information about the webhook payload for run start , see the `PlaybookRunWebhookPayload struct <https://github.com/mattermost/mattermost-plugin-playbooks/blob/b4c8058d8660efe35050bc7eb080e3819c7ab09c/server/app/playbook_run_service.go#L176-L185>`_. An example of the JSON payload for a run start `is available here <https://gist.github.com/icelander/b68f2bf2b4ffefec93400cb050211cf1>`_.
```

Comment:
nit, space before comma:
```suggestion
- For information about the webhook payload for run start, see the `PlaybookRunWebhookPayload struct <https://github.com/mattermost/mattermost-plugin-playbooks/blob/b4c8058d8660efe35050bc7eb080e3819c7ab09c/server/app/playbook_run_service.go#L176-L185>`_. An example of the JSON payload for a run start `is available here <https://gist.github.com/icelander/b68f2bf2b4ffefec93400cb050211cf1>`_.
- ```

Reply:
Both of these permalinks feel like they will quickly become stale, despite their intent to be otherwise. This is fine to merge, but can you help create a follow-up ticket to document this more explicitly in our code that will serve as a better permalink target?

Reply:
Great point; will do.